cmd/codesign.go

```diff
@@ -0,0 +1,53 @@
+// Copyright © 2016 Prometheus Team
```

Comment:
New file needs current year.
```suggestion
// Copyright © 2024 Prometheus Team
```

Reply:
Updated to 2024